Log/Parser/Piwik.php

```diff
@@ -16,6 +16,21 @@ class Piwik implements Parser
 {
     public function parse(Line $line)
     {
+
+        /*
+        * First try to json decode the message in case json logging is enabled
+        */
+        $message = json_decode($line->content);
+
+        if (is_object($message)) {
+            $message->severity = $message->level;
+            return $message;
+        }
+
+        /*
+            Here json logging is disabled -OR- we are parsing 
```

Comment:
Fix the failing PHPCS test
```suggestion
            Here json logging is disabled -OR- we are parsing
```